blockproducer/metastate.go

```diff
@@ -903,9 +888,13 @@ func (s *metaState) updatePermission(tx *types.UpdatePermission) (err error) {
 
 	// check whether sender is admin and find targetUser
 	isAdmin := false
+	numOfAdmin := 0
 	targetUserIndex := -1
 	for i, u := range so.Users {
 		isAdmin = isAdmin || (sender == u.Address && u.Permission == types.Admin)
+		if u.Permission == types.Admin {
+			numOfAdmin += 1
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
should replace numOfAdmin += 1 with numOfAdmin++